qiskit/transpiler/passmanager.py

```diff
@@ -26,18 +28,27 @@
 
 
 class PassManager:
-    """A PassManager schedules the passes"""
+    """Manager for a set of Passes and their scheduling during transpilation."""
 
-    def __init__(self, passes=None, max_iteration=1000, callback=None):
-        """Initialize an empty PassManager object (with no passes scheduled).
+    def __init__(
+            self,
+            passes: Union[BasePass, List[BasePass]] = None,
+            max_iteration: int = 1000,
+            callback: Callable = None
+    ):
+        """Initialize an empty `PassManager` object (with no passes scheduled).
 
         Args:
-            passes (list[BasePass] or BasePass): A pass set (as defined in ``append()``)
-                to be added to the pass manager schedule. The default is None.
-            max_iteration (int): The schedule looping iterates until the condition is met or until
-                max_iteration is reached.
-            callback (func): DEPRECATED - A callback function that will be called after each
-                pass execution.
+            passes: A pass set (as defined in ``append()``) to be added to the pass manager
```

Comment:
I think this would also work, if you wanted, but leaving it as is would also be good.
```
:py:func:`qiskit.transpiler.PassManager.append`
```
(I could be getting the import path wrong)

Reply:
Sure! Added at 47450e4a8c175f48da30878a7b3d0bfa504fc912